models/auth/auth_token.go

```diff
@@ -54,6 +54,11 @@ func DeleteAuthTokenByID(ctx context.Context, id string) error {
 	return err
 }
 
+func DeleteAuthTokensByUserID(ctx context.Context, uid int64) error {
+	_, err := db.GetEngine(ctx).Where(builder.Eq{"user_id": uid}).Delete(&AuthToken{})
```

Comment:
Don't we also need this check when deleting a user (`services/user/delete.go#75`)?

Reply:
Yes, good addition. Otherwise the tokens will just expire and get cleaned up after that.